functional-tests/fn_prover_checkpoint_runner.py

```diff
@@ -0,0 +1,19 @@
+import time
+
+import flexitest
+
+
+@flexitest.register
+class ProverClientTest(flexitest.Test):
+    def __init__(self, ctx: flexitest.InitContext):
+        ctx.set_env("prover")
+
+    def main(self, ctx: flexitest.RunContext):
+        # Wait for the Prover Manager setup
+        time.sleep(60)
+
+        # Test on with the latest checkpoint
+        print("Waiting for checkpoint runner")
+
+        time_out = 10 * 60
+        time.sleep(time_out)
```

Comment:
Is it todo? what is it supposed to do?

Reply:
This was just used for spawning up the prover client and manually checking the log file.